files/healthcheck.sh

```diff
@@ -16,30 +15,15 @@ function cfn-signal() {
 }
 trap cfn-signal EXIT
 
-sleep 5 # It takes compose a few seconds to create the container
-
-CONTAINER_NAME=$(/opt/bin/docker-compose -f /home/core/docker-compose.yml ps | head -n 3 | tail -n 1 | cut -f 1 -d" ")
-
-# If no healthcheck has been defined, use regular container status
-if [[ $(docker inspect ${CONTAINER_NAME} | jq '.[].State.Health|length') == 0 ]]; then
-  while [ "$RETRIES" -le "$MAX_RETRIES" ]; do
-    echo '.'
-    if [[ $(docker inspect ${CONTAINER_NAME} | jq '.[].State.Status') == '"running"' ]]; then
-      exit 0
+if [ ${USING_ELB} == 'True' ]; then
+    until [ "$state" == "\"InService\"" ]; do state=$(/usr/bin/docker run --rm xueshanf/awscli aws elb describe-instance-health --region ${AWS_REGION} --load-balancer-name ${STACK_NAME} --instances $(curl -s http://169.254.169.254/latest/meta-data/instance-id) --query InstanceStates[0].State); sleep ${SLEEP}; done
+else
+    sleep ${SLEEP} # It takes compose a few seconds to create the container
```

Comment:
@fiunchinho I think that ELB sleep should be (HealthCheck Interval) * (Healthy Threshold) as least.
By default 5s maybe not always is enough to put instances healthy